src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -6,28 +6,25 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.apache.mesos.MesosSchedulerDriver;
+import org.apache.mesos.Protos;
```

Comment:
There are methods used from org.apache.mesos.Protos so that is why it's included.